charts/gadget/values.yaml

```diff
@@ -2,6 +2,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Labels to be added to all other resources.
```

Comment:
nit: It is missing `--` prefix in order to auto generate documentation. (Also, I would like it below config object)